spring-boot/src/main/java/org/springframework/boot/context/embedded/tomcat/TomcatEmbeddedServletContainerFactory.java

```diff
@@ -223,6 +232,57 @@ protected void prepareContext(Host host, ServletContextInitializer[] initializer
 		postProcessContext(context);
 	}
 
```

Comment:
Please note that the API differences between Tomcat8 and Tomcat7 forced me to use reflection in order to support both web servers.